### ISSUE TITLE
Feature request: mpl_get_parsim_t

### ISSUE THREAD

**Issue by ms609** [mbrazeau/MorphyLib#13]
It'd be handy for debugging if mpl.c contained a function allowing one to check the parsim_t applied to a given character. Would that be easy to implement?

**Comment by mbrazeau** [comment]
Yes. Easy enough to do. I can try to add it today if I get time.

**Comment by mbrazeau** [comment]
I've also updated wagner, but didn't yet merge it to master because the Wagner functions aren't fully working yet.
